Tests/CommandsTests/APIDiffTests.swift

```diff
@@ -317,7 +318,46 @@ final class APIDiffTests: XCTestCase {
                     XCTFail("Unexpected error")
                     return
                 }
-                XCTAssertTrue(stderr.contains("error: Couldn’t check out revision ‘7.8.9’"))
+                print(stderr)
```

Comment:
Is this for debugging?

Reply:
yeah, I left that in by accident, thanks!